apps/condo/domains/ticket/schema/GetAllResidentTicketsService.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Generated by `createservice ticket.GetAllResidentTicketsService --type queries`
+ */
+const { Ticket } = require('../utils/serverSchema')
+const { GQLCustomSchema } = require('@core/keystone/schema')
+const access = require('@condo/domains/ticket/access/GetAllResidentTicketsService')
+const { mapTicketToResidentTicket } = require('../utils/serverSchema')
+
+const GetAllResidentTicketsService = new GQLCustomSchema('GetAllResidentTicketsService', {
```

Comment:
U don't need to create separate serivce for getting the list of tickets

Reply:
The `createservice` utility can create two types of services: mutations and queries.  I thought it would be more correct to separate them, isn't it?